src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/eltwise.cpp

```diff
@@ -251,6 +251,7 @@ std::string EltwiseLayerCPUTest::getPrimitiveType(const utils::EltwiseTypes& elt
 }
 
 TEST_P(EltwiseLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
```

Comment:
Could you please elaborate a little bit on the purpose of inserting this Macro? The thing is that this check is called in the base class: https://github.com/openvinotoolkit/openvino/blob/c0908b3f14e5938de4f86e31470ec65709a9b2ca/src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp#L44-L46
So why do we need inserting this Macro here and in all the other similar places?